### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Farzannajipour**
Correcting misspelled words

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
 adaptive media formats (such as [DASH][] and [HLS][]) in a browser, without
-using plugins or Flash.  Instead, Shaka Player uses the open web standards
+using plugins or Flash.  Instead, Shaka Player uses open web standards
 [MediaSource Extensions][] and [Encrypted Media Extensions][].
 
-Shaka Player also supports [offline storage and playback][] of media using
-[IndexedDB][].  Content can be stored on any browser.  Storage of licenses
+Shaka Player also supports [offline storage and playback][] of media use
+[IndexedDB][].  Content can be stored on any browser.  The storage of licenses
```
Comment (Farzannajipour) [inline README.md]: correcting use & the storage

File: README.md [hunk 2]
```diff
@@ -26,35 +26,35 @@ For details on what's coming next, see our [development roadmap](roadmap.md).
...
-We support iOS through Apple's native HLS player.  We provide the same top-level
-API, but we just set the video's `src` element to the manfiest/media.  So we are
+ - ¹: Only the latest stable version is tested and supported. Older releases may still be usable, and we will accept pull requests for them, but they will not be officially tested or supported.
+ - ²: The latest stable Chromecast firmware is tested. Both the sender and receiver can be implemented with Shaka Player.
+ - ³: Tizen 2017 model is actively tested and supported by the Shaka Player team. Tizen 2016 model is community-supported and untested by us.
+ - ⁴: IE 11 offers PlayReady support on Windows 8.1 and Windows 10 only. IE 11 can play clear content on Windows 8.0. IE 11 does not support adaptive playback on Windows 7 and under.
+ - ⁵: These are expected to work, but are not actively tested by the Shaka Player team.
+
+We support iOS through Apple's native HLS player.  We provide the same top-level
+API, but we just set the video's `src` element to the manifest/media.  So we are
```
Comment (Farzannajipour) [inline README.md]: correcting manifest name

File: README.md [hunk 3]
```diff
@@ -74,88 +74,88 @@ You can also create a [manifest parser plugin][] to support custom manifest form
...
+ - I-frame-only playlists: https://github.com/google/shaka-player/issues/742
+ - Raw AAC (without an MP4 container): https://github.com/google/shaka-player/issues/1083
 
 [mux.js]: https://github.com/videojs/mux.js/releases
 
 
 ## DRM support matrix
 
-|Browser   |Widevine  |PlayReady|FairPlay |ClearKey⁶ |
+|Browser   |Widevine  |PlayReady|FairPlay |ClearKey⁶ |
```
Comment (joeyparrish) [inline README.md]: I don't see what you've changed in the tables.  From GitHub, it appears that you may have added or removed a single space, but it's hard to tell.  Please avoid reformatting whitespace.  It makes it very hard to spot the meaningful, non-cosmetic changes.

File: README.md (hunk 2 above)
Comment (joeyparrish) [reply README.md]: Thank you.  This was a definite typo.

File: README.md [hunk 4]
```diff
@@ -26,35 +26,35 @@ For details on what's coming next, see our [development roadmap](roadmap.md).
...
- - ²: Latest stable Chromecast firmware is tested. Both sender and receiver can be implemented with Shaka Player.
- - ³: Tizen 2017 model is actively tested and supported by the Shaka Player team. Tizen 2016 model is community-supported and untested by us.
- - ⁴: IE 11 offers PlayReady support on Windows 8.1 and Windows 10 only. IE 11 can play clear content on Windows 8.0. IE 11 does not support adaptive playback on Windows 7 and under.
- - ⁵: These are expected to work, but are not actively tested by the Shaka Player team.
-
-We support iOS through Apple's native HLS player.  We provide the same top-level
-API, but we just set the video's `src` element to the manfiest/media.  So we are
+ - ¹: Only the latest stable version is tested and supported. Older releases may still be usable, and we will accept pull requests for them, but they will not be officially tested or supported.
+ - ²: The latest stable Chromecast firmware is tested. Both the sender and receiver can be implemented with Shaka Player.
```
Comment (joeyparrish) [inline README.md]: Thank you.  This change can stay, as well.

File: README.md (hunk 1 above)
Comment (joeyparrish) [reply README.md]: "Storage" is a valid subject in English without the need for "the", but "The storage of licenses" arguably sounds a little better.  So that can stay.

The other change just above should be reverted.  "Shaka supports storage of media **using** IDB" is correct.  "Shaka supports storage of media **use** IDB" does not make sense in English.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 # ![Shaka Player](docs/shaka-player-logo.png)
 
-Shaka Player is an open-source JavaScript library for adaptive media.  It plays
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
 adaptive media formats (such as [DASH][] and [HLS][]) in a browser, without
-using plugins or Flash.  Instead, Shaka Player uses the open web standards
+using plugins or Flash.  Instead, Shaka Player uses open web standards
```
Comment (joeyparrish) [inline README.md]: The word "the" was correct in the phrase "uses the open web standards MSE & EME".  Please revert this change.

File: README.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 # ![Shaka Player](docs/shaka-player-logo.png)
 
-Shaka Player is an open-source JavaScript library for adaptive media.  It plays
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
```
Comment (joeyparrish) [inline README.md]: Please revert all whitespace changes.  These obscure the real corrections and changes.

**Comment by Farzannajipour** [comment]
@googlebot I signed it!
